docs/UserGuide.md

```diff
@@ -110,27 +112,27 @@ Tags: FallRisk, Diabetes
 You now have 1 patient(s) in your address book.
 ```
 
-### Listing all patients : `list`
+### Listing patients : `list`
+
+Displays a list of patients in NAB. 
 
-Displays a list of all registered patients.
+By using list with optional parameters, you will be able to list all patients.
 
-Format: `list`, `list [w\WARD] [t\TAG]...`
+Optionally, you can filter by ward or tags to list only patients in a specific ward or with specific tags.
+
+Format: `list [w\WARD] [t\TAG]...`
 
 * WARD: Must only list at most 1 ward.
 
-Example command:
 
 `list`
 ```
-Here are the details of the 2 patients in your contact book:
-
-1. John Doe
-2. Jane Doe
+Listed all persons
```

Comment:
Perhaps the output for the example command of `list` should actually show the whole list?

Reply:
I have amended, does the change look good?